src/api/java/com/minecolonies/api/util/BlockPosUtil.java

```diff
@@ -735,4 +737,97 @@ public static BlockPos findAround(final IBlockReader world, final BlockPos start
 
         return null;
     }
+
+    /**
+     * Returns plane furthest from lookFrom within the box.
+     *
+     * @param lookFrom from where to look at the box
+     * @param boxStart possibly unsorted box start pos
+     * @param boxEnd   possible unsorted box end pos
+     * @param ignoreY  if should ignore Y planes or not
+     * @return furthest plane from lookFrom within the box
+     */
+    public static Direction getBoxFurthestPlane(final BlockPos lookFrom,
```

Comment:
I would really love to know if there is a better way to do this
(btw this doesnt do the common "sort plane centers based on lookFrom distance")